BetterPasswordPlugin.inc.php

```diff
@@ -304,4 +377,27 @@ function addUserSettings($hookname, $args) {
 		);
 		return false;
 	}
+        /**
+	 * Callback to fill cache with data, if empty.
+	 * @param $cache FileCache
+	 * @param $password_hash string The leading two characters of the hash users password
+	 * @return $cache_password array The final cache of all the passwords corresponding to hash created by user password
```

Comment:
This return value should now just be a string.

Reply:
The value is Boolean since in_array() returns true or false ?

Reply:
Yes, you are right.